source/code/source/playfab/PlayFabEventPipeline.cpp

```diff
@@ -216,7 +220,7 @@ namespace PlayFab
                 // give some time back to CPU, don't starve it without a good reason
                 std::this_thread::sleep_for(std::chrono::milliseconds(this->settings->readBufferWaitTime));
             }
-            catch (const std::exception& ex)
+            catch (const std::exception & ex)
```

Comment:
Is this done for an explicit reason? I usually see no space between a reference and the object it refers to. It doesn't change anything according to the compiler, but it's just visual consistency asthetics.

Reply:
VS auto spacing default difference between VS 17 & 19
Will fix

---
In reply to: [360618016](https://github.com/PlayFab/XPlatCoreTemplate/pull/69#discussion_r360618016) [](ancestors = 360618016)